src/Page/Guide/View.elm

```diff
@@ -60,6 +87,7 @@ viewImageColumn guide =
             Nothing ->
                 text ""
         ]
+    , div [] [ text (t InCategory ++ t (categorySlugToKey guide.categorySlug)) ]
```

Comment:
This will probably need to move but fine for now

Reply:
This is very much my take on it